content/2016/11/04/branding-type.rst

```diff
@@ -0,0 +1,186 @@
+.. _Sondra: http://localhost:3000/authors/sondra/
+.. _Stacy: http://localhost:3000/authors/stacy/
+
+
+Element Collages, Moodboards, and Style Tiles? Oh my!
+-----------------------------------------------------
+
+
+At our fall work retreat, the full OddBird team met with a business development professional who helped us further clarify and confirm our brand goals. The design team spent time individually looking at the brand elements of our favorite websites and sites with similar goals. We came back together for a show-and-tell, first among ourselves, and then with a brief overview of our findings and ideas for the full team. That’s when we hit the first hurdle.
+
+Do we create element collages, put together mood boards, or create style tiles to take these general brand ideas to the next stage? We each had successful experiences using all of these methods to create brands for clients, and we went back and forth about which one would work best for us.
+
+**Mood Boards:**
+A `mood board`_ is a collection of elements such as text, images, colors, textures, and styles that convey a particular look and feel or mood for a web project. Talking through several mood boards with a client can be an excellent place to start when you need to pick a fairly general brand direction.
+
+**Style Tiles:**
+The genius of style tiles is the way they define a middle
+ground — more specific than a mood board and more open for
+interpretation than a full mockup. Samantha Warren wrote an excellent article for A List Apart called `Style Tiles and How They Work`_.
+
+**Element Collages:**
+Element collages focus on brand design for a set of elements grouped together, a form, for example. Element collages are quicker to design than full mockups, but detailed enough to help a client visualize an implemented brand. Daniel Mall goes into detail on `Element Collages`_ on his blog.
+
+The design team struggled with which technique would be most useful for our internal design process. There were too many elements to consider at once in our element collages, and we were loath to use the blender method of mixing and matching, but neither did we feel that any one element collage encapsulated our brand. We tried style tiles, but those felt both too specific and not specific enough.
+
+.. image:: /static/images/blog/2016/brand/type-styletiles.jpg
+
+*Style Tiles - 4 cropped sections*
+
+.. image:: /static/images/blog/2016/brand/type-elcollage.jpg
+
+*Element Collage*
+
+.. image:: /static/images/blog/2016/brand/type-elcollage2.jpg
+
+*Element Collage*
+
+Sometimes tools that worked before, don't work on a new team or project. Fortunately, we each had enough experience with creative work to realize that getting stuck isn't a failure, in fact, it is an expected and valuable part of the process. The goal isn’t to stop getting stuck, but to recognize it quickly and know how to get unstuck. It’s a clue that you need to step back, look around, and try something new.
```

Comment:
Better. `recognizing the situation for what it is and know how to get unstuck` is still confusing to me. Maybe...

```
Good process doesn't mean ideas always flow smoothly, without ever going down the wrong path. Our goal isn't to foresee and avoid every problem, but to recognize when we're getting stuck, and have the available tools to try a different approach.
```

Or something like that? Partially it's just a matter of getting the phrase tenses to match up, and partly trying to dig into the core of what this means — less about knowing, and more about having flexibility to try different things. What do you think?